test/unittests/NDB_BVL_Instrument_LINST_ToJSON_Test.php

```diff
@@ -60,7 +60,12 @@ function _getAllMethodsExcept($methods) {
     function testMetaData() {
         $instrument = "table{@}Test\ntitle{@}Test Instrument";
         $base64 = "data://text/plain;base64," . base64_encode($instrument);
-        $this->i->loadInstrumentFile($base64, true);
+        try {
```

Comment:
since these try-catch blocks are not important for the test. why don't you remove it?
it always catches the exception and makes loadInstrumentFile function failed.

Reply:
I think before I added this a different error occurred `NotFound`.

I'll delete it and we'll see what happens.

Reply:
@kongtiaowang Hi Shen, are you saying we need to remove all the try catch blocks?

Reply:
@sruthymathew123 yes, delete them.

Reply:
@kongtiaowang Shen, I re added the try catch blocks again as per your suggestion. Can you close this conversation if it is all good.